samples/c2pa/C2paPlayer/C2paMenu/C2paMenuFunctions.js

```diff
@@ -98,16 +112,43 @@ export let updateC2PAMenu = function (c2paStatus, c2paMenu , isMonolithic , vide
         );
 
         if (c2paItemValue != null) {
+            //formatting for social media links
+            if(c2paItemKey=== 'SOCIAL' ){
+                var socialArray = c2paItemValue.map(function (account) {
+                    var formattedWebsite = providerInfoFromSocialId(account).name
+                    return `<span><a class="url" href=${account}>${formattedWebsite}</a></span>`
+         
+                });
+                c2paItem.innerHTML =
+                '<span class="itemName nextLine">' +
+                c2paItemName +
+                '</span>' +
+                c2paMenuInstance.c2paMenuDelimiter() + socialArray.join('\n')
+            }
             //If the value is not null, we update the menu item text and show it
-            c2paItem.innerText =
+            else if (c2paItemValue.length >= 32) {
```

Comment:
@dkozma Is there a 'check' for truncation I can use without using a character value? I want strings of a certain length to automatically go on the second line

Reply:
We shouldn't use length to determine truncation if possible, we should really use [CSS](https://css-tricks.com/snippets/css/truncate-string-with-ellipsis/). For example:
OOOOOOOOOO vs
IIIIIIIIII.
Is that an option here?

Reply:
Let's talk when you get a chance.